src/mngt/ocf_mngt_core.c

```diff
@@ -21,6 +21,12 @@ static ocf_seq_no_t _ocf_mngt_get_core_seq_no(ocf_cache_t cache)
 	return ++cache->conf_meta->curr_core_seq_no;
 }
 
+static int ocf_mngt_core_set_name(ocf_core_t core, const char *name)
+{
+	return env_strncpy(core->conf_meta->name, OCF_CORE_NAME_SIZE - 1,
+			name, env_strnlen(name, OCF_CORE_NAME_SIZE));
```

Comment:
Same here, looks like env_strnlen() doesn't add anything on top of OCF_CORE_NAME_SIZE

Reply:
Done.

Reply:
I meant "OCF_CORE_NAME_SIZE - 1" , not "OCF_CORE_NAME_SIZE"

Reply:
As it's guaranted that `env_strncpy()` always appends null terminator, changed both source and destination lengths to `OCF_CORE_NAME_SIZE`.